connect/src/com/telenor/connect/WellKnownAPI.java

```diff
@@ -28,5 +33,44 @@ public String getIssuer() {
         public Set<String> getNetworkAuthenticationTargetIps() {
             return networkAuthenticationTargetIps;
         }
+
+        protected WellKnownConfig(Parcel in) {
+            issuer = in.readString();
+            networkAuthenticationTargetIps = new HashSet<>();
+            int count = in.readInt();
```

Comment:
Mini tip: If you switch these two lines you can initialize the hash set to the right size. No benefit for small sets though.